system/TEMPLATE/etc/httpd/conf.d/cfme-https-cert.conf

```diff
@@ -1,6 +1,7 @@
 ## CFME SSL Virtual Host Context
 
 <VirtualHost *:8443>
+KeepAlive on
```

Comment:
Why is the other one a capitalized "On", and this one is lowercased?  Does it matter?